frontend/src/pages/sport items/leaguePage.js

```diff
@@ -113,18 +157,57 @@ function LeaguePage() {
             )
     }, [loading]);
 
+    const initialIcon = <FaFilter size={28} />
+
+    const [selectedModel, setSelectedModel] = useState("leagues");
+    const [filters, setFilters] = useState([]);
+    const [burgerMenu, setBurgerMenu] = useState(false)
+    const [menuIsOpen, setMenuIsOpen] = useState(false)
+    const [menuIcon, setMenuIcon] = useState(initialIcon)
+
+    useEffect(() => {
+
+        const handleResize = () => {
+            const smallScreen = window.innerWidth <= 1050
```

Comment:
MINOR:

Maybe we should also have this one moved somewhere.

That'd allow us to make sort of a "standard" to follow (all supported screens, for example)